packages/react-components/react-combobox/src/components/OptionGroup/OptionGroup.test.tsx

```diff
@@ -20,10 +20,22 @@ describe('OptionGroup', () => {
     },
   });
 
-  // TODO add more tests here, and create visual regression tests in /apps/vr-tests
-
   it('renders a default state', () => {
     const result = render(<OptionGroup>Default OptionGroup</OptionGroup>);
     expect(result.container).toMatchSnapshot();
   });
+
+  it('renders with a label', () => {
+    const result = render(<OptionGroup label="optgroup label">Default OptionGroup</OptionGroup>);
+    expect(result.container).toMatchSnapshot();
```

Comment:
We should eventually move off snapshot tests for all components (not for this PR but for later consideration).